### DISCUSSION TITLE
Welcome to Helios Discussions!

### DISCUSSION THREAD

**Discussion by derammo** [HeliosVirtualCockpit/Helios#565] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by derammo** [comment, marked as answer]
Works.

**Comment by HolgerBanski** [comment]
Hi, is it possible to move all modifications that Helios needs to the saved games folder? Since ED introduced 'clean script' for servers, I cannot connect to protected servers anymore, because my scripts are 'tainted'. The viewport changes are in the DCS game folder and seem to cause this problem.

**Reply by derammo** [reply]
No, this is not possible.  Specifically, the point of those "clean script" rules are to enforce that people run unmodified DCS.   With the viewports added, it isn't unmodified anymore.   

I had a chat with Eagle Dynamics in the past and I told them exactly how we use viewport mods and that we'd be happy to work with them to get certain changes whitelisted, but then they decided not to pursue it.   Helios is "somewhat modified DCS" when it comes to integrity check and that is just the truth.

**Reply by HolgerBanski** [reply]
Thank you for your response, I feared that would be the answer. Eagle Dynamics should be happy that people spend their time to improve their application, providing features they can't or won't be able to provide. I understand that this is a modification, but it should be whitelisted. Otherwise somebody would have to enforce as well, that everybody plays without VR/head-/eye-tracking, without hotas and on a 27" HD screen, because someone out there has exactly that setup and would have a disadvantage. But yeah, that is ED and not Helios' decision/fault. Thank you for the great work and making it possible to actually build a simpit!

**Comment by MaxGuillen** [comment]
Hi, when there's going to be a new release?

**Comment by MaxGuillen** [comment]
Do you need collaborators? is this a visual studio solution? Could you tell me whats the architectuer of this solution I might help from time to time, is it a C# winforms app?

**Comment by pwilliamssa** [comment]
I am looking at making panels for the F/A-18 and using Helios for the Integrated Fuel / Engine Indicator and the Backup instruments / RWR.  I have the WIN WIN DDI's.  My question is will Helios and WIN WIN software get along or will they interfere with each other?

**Comment by gdpalumbo** [comment]
Is there any way we can get fuel flow and fuel gauge for the f16? The fuel flow is there as an f16 item but doesn’t not function, captain zee got this to function in his own way, but I don’t want to use his profile. Please please can we get these two items in  and working? It’s a big thing for a lot of us.

**Reply by Todd1215** [reply]
I assume this is a DCS problem with the Fuel Flow Guage. I can see that the F-16C DCS Interface splits out all the values 10k, 1k, and 100's place. Probably CaptZeen used a function in his interface driver to create a total value to send to the FF control. Are you looking for a FF gauge and Fuel gauge that do all this for you? You should open an enhancement request so that it gets on the list of enhancements.

**Comment by NoLegalMoves129** [comment]
I am new to Helios so would like some help in determining whether my use case is supported. I've worked on it for a couple of weeks and ready to give up at this point..
I have a triple 4k TV cockpit view and an additional 3 small screens for MFCDs (for the FA/18)
I tried configuring Helios with or without Nvidia surround and variations of various resolutions. What works is triple screen (triple 4k) with the '3 screens' option in DCS. I managed to setup the 4 MFCDs and display them in game but it's always blacking out my right most 4k screen and compressing the view into the 2 left most TVs. Any use cases identical to mine out there that do work? (My research online always only yields simplified, lower resolution examples)

**Comment by gdpalumbo** [comment]
How do I open an enhancement request? Sorry new to GitHub. And yes it would be nice if we could add a fuel garage and fuel flow like we can add all the other gauges and bind a telemetry item to it like we do the rest. I’d assume this is a pretty common request for this, we all want it lol

Thank You,

Gino Palumbo
Vice President
Rock-Vern Electric Inc

24 Tolland Stage Road
Tolland, CT 06084
Phone: (860) 875-5905
Fax: (860) 875-6111
Cell: (860) 833-4687



________________________________

**Comment by gdpalumbo** [comment]
Gauges are already on Helios, they just don't work

Thank You,

Gino Palumbo
Vice President
Rock-Vern Electric Inc

24 Tolland Stage Road
Tolland, CT 06084
Phone: (860) 875-5905
Fax: (860) 875-6111
Cell: (860) 833-4687



________________________________

**Comment by Todd1215** [comment]
At the top of this page click on Issues, then click the green button to the right that Says "New Issue". Provide details as to what the issue is. You need to do this so someone can look into it. Please provide as much detail as you can. Don't just open an issue that says "this doesn't work". I have added an enhancement request #811 for the fuel gauge to be created.
<img width="1487" alt="Screen Shot 2024-02-27 at 3 10 33 PM" src="https://github.com/HeliosVirtualCockpit/Helios/assets/6605805/4fea1ad6-60e8-4ce5-96db-5e4927c5a398">
